client/src/Components/NewAlbum.js

```diff
@@ -1,12 +1,22 @@
-import { useState } from "react";
-import { Box, Button, Text, Layer, TextInput, DateInput } from "grommet"
+import { useState, useEffect } from "react";
+import { Box, Button, CardHeader, Layer, Card, Select, Text, TextInput, DateInput } from "grommet"
 import { FormClose } from "grommet-icons";
 
 const NewAlbum = ({onClose, getAlbums}) => {
     const [date, setDate] = useState();
     const [artist, setArtist] = useState("");
     const [album, setAlbum] = useState("");
     const [url, setUrl] = useState("");
+    const [genre, setGenre] = useState("");
+    const [options, setOptions] = useState([]);
+
+    const getGenre = () => fetch("http://localhost:4000/genre")
+                        .then(res => res.json())
+                        .then(data => setOptions(data));
+
+    useEffect(() => {
+        getGenre();
+        }, [])
 
     const onChange = (event) => {
```

Comment:
get rid of this handler, do it the same way as you do it with text fields "Album" or "Artist"